app/models/job_invocation.rb

```diff
@@ -48,7 +48,7 @@ class JobInvocation < ActiveRecord::Base
 
   scope :with_task, -> { references(:task) }
 
-  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :auto_complete => true
+  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :complete_value => true
```

Comment:
tbh: I dont know if it should be true (e.g. is there a value to show the auto completer id?)